### PR TITLE
javascript-data-model | Rich Trunzo

### PR DESCRIPTION
**Pull request by richtrunzo**
After pushing I realize I'm missing a comma on the second to last object. I can fix it and re push if helpful.

Screen Shot 2020-11-17 at 10 49 37 AM
Screen Shot 2020-11-17 at 10 49 46 AM

### CODE REVIEW

File: javascript-data-model/main.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    orderplaced: "July 4, 2020",
+    total: 17.22,
+    price: 15.98,
+    shipto: "JS Masher",
+    ordernumber: "114-2875557-9059409",
+    delivered: "July 7, 2020",
+    title: "Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter",
+    author: undefined
+  },
+  [
```
Comment (BrettAlbright) [inline javascript-data-model/main.js]: I would recommend using a single object here for the order, and then an array for one of the properties which contains the two objects which represent the items ordered

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update

File: javascript-data-model/main.js (hunk 1 above)
Comment (richtrunzo) [reply javascript-data-model/main.js]: Thanks Brett, updated per your feedback - is this correct?

Screen Shot 2020-11-17 at 11 00 22 AM

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**